error_pages/404.php

```diff
@@ -2,118 +2,131 @@
+		This message may not be removed from any pages based on the UNLedu Web Framework.
 
-	$Id: php.fixed.dwt.php | 6edb0e1ee94038935f3821c6ce15dfd5c217b2e2 | Tue Dec 1 17:08:56 2015 -0600 | Kevin Abel  $
--->
-<?php wdnInclude("/wdn/templates_4.1/includes/scriptsandstyles.html"); ?>
-<!-- TemplateBeginEditable name="doctitle" -->
-<title>Not Found | VITA Lincoln | University of Nebraska&ndash;Lincoln</title>
-<!-- TemplateEndEditable -->
-<!-- TemplateBeginEditable name="head" -->
-<link rel="stylesheet" href="/dist/error_pages/error_page.css">
-<!-- TemplateEndEditable -->
-<!-- TemplateParam name="class" type="text" value="" -->
+		$Id: php.fixed.dwt.php | cf0a670a0fd8db9e20a169941c55c838d7c2ba10 | Wed Dec 12 16:54:41 2018 -0600 | Eric Rasmussen	$
+	-->
+	<!-- TemplateBeginEditable name="doctitle" -->
+	<title>Not Found | VITA Lincoln | University of Nebraska&ndash;Lincoln</title>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/head-2.html"); ?>
+	<!-- TemplateBeginEditable name="head" -->
+	<link rel="stylesheet" href="/dist/error_pages/error_page.css">
+	<!-- TemplateEndEditable -->
+	<!-- TemplateParam name="class" type="text" value="" -->
 </head>
-<body class="hide-wdn_identity_management" data-version="4.1">
-	<?php wdnInclude("/wdn/templates_4.1/includes/skipnav.html"); ?>
-	<div id="wdn_wrapper">
-		<input type="checkbox" id="wdn_menu_toggle" value="Show navigation menu" class="wdn-content-slide wdn-input-driver" />
-		<?php wdnInclude("/wdn/templates_4.1/includes/noscript-padding.html"); ?>
-		<header id="header" role="banner" class="wdn-content-slide wdn-band">
-			<div id="wdn_header_top">
-				<span id="wdn_institution_title"><a href="https://www.unl.edu/">University of Nebraska&ndash;Lincoln</a></span>
-				<div id="wdn_resources">
-					<?php wdnInclude("/wdn/templates_4.1/includes/wdnResources.html"); ?>
-					<?php wdnInclude("/wdn/templates_4.1/includes/idm.html"); ?>
-					<?php wdnInclude("/wdn/templates_4.1/includes/search.html"); ?>
-				</div>
-			</div>
-			<div id="wdn_logo_lockup">
-				<div class="wdn-inner-wrapper">
-					<?php wdnInclude("/wdn/templates_4.1/includes/logo.html"); ?>
-						<span id="wdn_site_affiliation"><!-- TemplateBeginEditable name="affiliation" --><!-- TemplateEndEditable --></span>
-						<span id="wdn_site_title"><!-- TemplateBeginEditable name="titlegraphic" -->VITA Lincoln<!-- TemplateEndEditable --></span>
-				</div>
-			</div>
-		</header>
-		<div id="wdn_navigation_bar" class="wdn-band">
-			<nav id="breadcrumbs" class="wdn-inner-wrapper" role="navigation" aria-label="breadcrumbs">
-				<!-- TemplateBeginEditable name="breadcrumbs" -->
-				<ul>
-					<li><a href="https://www.unl.edu/" title="University of Nebraska&ndash;Lincoln" class="wdn-icon-home">UNL</a></li>
-					<li><a href="/" title="VITA Lincoln">VITA Lincoln</a></li>
-					<li>Not Found</li>
-				</ul>
+<body class="@@(_document['class'])@@ unl" data-version="5.0">
```

Comment:
What is this whole `@@(_document['class'])@@` stuff?

Reply:
Most of this stuff came from the new template files.

https://github.com/unl/wdntemplates/blob/4767842e3efd6c537079a75481708ec7de7495a0/Templates/php.fixed.dwt.php#L28